src/test/skript/tests/syntaxes/expressions/ExprLastAttack.sk

```diff
@@ -0,0 +1,17 @@
+test "last known damage/attacker/cause":
+	clear all entities
+	spawn a zombie at spawn of world "world"
+	set {_zombie} to the last spawned zombie
+	spawn a pig at spawn of world "world":
+		assert last known attacker of event-entity is not set with "The last known attacker of the pig was present"
+		set last known attacker of event-entity to {_zombie}
+		assert last known attacker of event-entity is set with "Failed to set the attacker to the zombie"
```

Comment:
We ought to verify the last known attacker _is_ the zombie at this line and at future lines - this is the part that may currently fail.